### ISSUE TITLE
Suggestion: Automated generating of shortcuts

### ISSUE THREAD

**Issue by t0msk** [PhoenicisOrg/phoenicis#1443]
Ability to generate shortcuts of applications in container. I know that I can create shortcut in Library tab, but it is not automated like in POL 4, in Phoenicis you have to full whole form by yourself.

How should it behave?

For example if I select app in Library then there should be option (right click on it) or button that Create shortcut, after clicking on that button then I would create shortcut to that app.

And in POL 4 if you install some application then it will scan for all created exe files and it give you option to create shortcut from these exe files or some button if you click then it will scan whole Program Files directory in container for exe files.

**Comment by plata** [comment]
So you would basically want a list which suggests possible executables to create a shortcut?

**Comment by t0msk** [comment]
Yes (POL 4 works like this somehow) and button to create shortcut for selected application.

**Comment by plata** [comment]
I don't get the second part. If you select an app in the library, you have the shortcut already.

**Comment by plata** [comment]
The shortcut generation should do that already. If it doesn't work for you, open a new issue for this please.

**Comment by qparis** [comment]
Related with #1644
